sci-libs/gsl/gsl-2.8.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic toolchain-funcs
+
+DESCRIPTION="The GNU Scientific Library"
+HOMEPAGE="https://www.gnu.org/software/gsl/"
+SRC_URI="mirror://gnu/${PN}/${P}.tar.gz
+	https://github.com/Jamim/${PN}/commit/${P//./%2E}-cblas.patch -> ${P}-cblas.patch"
+
+LICENSE="GPL-3+"
+SLOT="0/$(ver_cut 1-2)"
```

Comment:
diff:

```diff
 LICENSE="GPL-3+"
-# Usually 0/${PV} but check
-SLOT="0/27"
+SLOT="0/$(ver_cut 1-2)"
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux"
 IUSE="cblas-external +deprecated static-libs"
```

I would assume e.g. @orlitzky would rather this comment stay here at least. Unsure whether the dot in the subslot matters to anyone. :)